src/main/java/org/hydev/mcpm/client/database/searchusecase/SearcherByKeyword.java

```diff
@@ -63,11 +63,13 @@ public List<PluginModel> getSearchList(String inp, List<PluginModel> plugins) {
         if (SearcherByKeyword.keywordMap == null) {
             SearcherByKeyword.keywordMap = constructSearchMaps(plugins);
         }
-        String [] keywords = inp.toLowerCase().split(" "); // Should be a string
-        Set<PluginModel> res = new HashSet<>(SearcherByKeyword.keywordMap.get(keywords[0]));
+        String [] keywords = inp.split(" "); // Should be a string
+        Set<PluginModel> res = new HashSet<>(SearcherByKeyword.keywordMap.getOrDefault(keywords[0], List.of()));
```

Comment:
why was `.toLowerCase()` moved outside this function? the keywords in the map are lowercase though...

Reply:
should be redundant bc the parser already converts inp to lowercase. Ill add the lower case conversion in searcher again to be safe.